src/gui/forcefieldtab_funcs.cpp

```diff
@@ -424,18 +426,20 @@ void ForcefieldTab::updatePairPotentialsTableRow(int row, PairPotential *pairPot
     item->setText(QString::number(pairPotential->chargeJ()));
 
     // Parameters
-    for (auto n = 0; n < MAXSRPARAMETERS; ++n)
+    auto col = 5;
+    for (auto x : pairPotential->parameters())
     {
         if (createItems)
         {
             item = new QTableWidgetItem;
             item->setData(Qt::UserRole, VariantPointer<PairPotential>(pairPotential));
             item->setFlags(Qt::ItemIsSelectable | Qt::ItemIsEnabled);
-            ui_.PairPotentialsTable->setItem(row, n + 5, item);
+            ui_.PairPotentialsTable->setItem(row, col, item);
         }
         else
-            item = ui_.PairPotentialsTable->item(row, n + 5);
-        item->setText(QString::number(pairPotential->parameter(n)));
+            item = ui_.PairPotentialsTable->item(row, col);
+        item->setText(QString::number(x));
+        ++col;
```

Comment:
```suggestion
            item = ui_.PairPotentialsTable->item(row, col++);
        item->setText(QString::number(x));
```

Same with these past two